pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -216,6 +222,25 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	return nil
 }
 
+func (vm *windows) ConfigureWindowsExporter() error {
```

Comment:
please add a doc comment for this method

Reply:
Method's doc located in Windows interface (line 85)